### ISSUE TITLE
Up/down keys cause un-handled errors to be thrown, version 2.1.3

### ISSUE THREAD

**Issue by vidhill** [ellipticaljs/paper-autocomplete#47]
Hello,

I installed newest version on this element, 

If there is nothing in the textbox, when you press the up or down keys it seems to be attempting to scroll through the suggestions list, 
It seems to me that is is due to the fact that there are no suggestions present that it is throwing errors 

For some reason I am getting different errors on my test project..

```
Uncaught TypeError: Cannot read property '2' of undefined
    at HTMLElement._moveHighlighted (paper-autocomplete-suggestions.html:680)
    at HTMLElement._onKeypress (paper-autocomplete-suggestions.html:628)
```

I pulled down your demo file and I am getting different errors, but that was on master so the code may be different

```
paper-autocomplete-suggestions.html:685 Uncaught TypeError: Cannot read property 'classList' of undefined
    at HTMLElement._moveHighlighted (paper-autocomplete-suggestions.html:685)
    at HTMLElement._onKeypress (paper-autocomplete-suggestions.html:631)

```

**Comment by jhuesos** [comment]
Ups, what a stupid mistake on my side... Apologies. It is an easy fix. I have something already working, I just need to test it in all browsers and write a unit test for it.

I will be delivering the fix before dinner time :) (if you happen to be in central europe time zone)

About the demo site... I really need to update that. The demo site is showing the component in a much older version. I am afraid to demo the latest version you need to clone and run the component locally.

**Comment by jhuesos** [comment]
Fixed in v2.1.4. Again, many thanks for the feedback. I will try to get the demo page updated soon.

**Comment by vidhill** [comment]
Thank you!
